src/timepicker/timepicker.component.ts

```diff
@@ -65,11 +73,28 @@ export class TimePicker {
 
 	@Output() valueChange: EventEmitter<string> = new EventEmitter();
 
+	writeValue(obj: any): void {
+		this.value = obj;
+	}
+	registerOnChange(fn: any): void {
+		this.onChangeHandler = fn;
+	}
+	registerOnTouched(fn: any): void {
```

Comment:
`onTouchedHandler` should be called when the component receives focus - at the moment it doesn't do anything